code/modules/mob/living/simple_animal/abnormality/he/eris.dm

```diff
@@ -0,0 +1,197 @@
+/mob/living/simple_animal/hostile/abnormality/eris
+	name = "Eris"
+	desc = "A towering, intimidating woman without a mouth."
+	icon = 'ModularTegustation/Teguicons/48x48.dmi'
+	icon_state = "eris"
+	icon_living = "eris"
+	maxHealth = 1100
+	health = 1100
+	ranged = TRUE
+	attack_verb_continuous = "claws"
+	attack_verb_simple = "claw"
+	stat_attack = HARD_CRIT
+	melee_damage_lower = 11
+	melee_damage_upper = 12
+	move_to_delay = 2.6
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.7, WHITE_DAMAGE = 1.3, BLACK_DAMAGE = 0.7, PALE_DAMAGE = 1)
+	speak_emote = list("croons")
+	pixel_x = -8
+
+	can_breach = TRUE
+	threat_level = HE_LEVEL
+	pet_bonus = TRUE
+	start_qliphoth = 3
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(35, 40, 40, 35, 35),
+						ABNORMALITY_WORK_INSIGHT = list(35, 40, 40, 35, 35),
+						ABNORMALITY_WORK_ATTACHMENT = 70,
+						ABNORMALITY_WORK_REPRESSION = list(50, 55, 55, 50, 45),
+						)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/coiling,
+		/datum/ego_datum/armor/coiling
+		)
+//	gift_type =  /datum/ego_gifts/coiling
+	abnormality_origin = ABNORMALITY_ORIGIN_ORIGINAL
+	var/girlboss_level = 0
+
+
+//Okay, but here's the breach on death
+/mob/living/simple_animal/hostile/abnormality/eris/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, .proc/on_mob_death) // Hell
+
+/mob/living/simple_animal/hostile/abnormality/eris/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/eris/proc/on_mob_death(datum/source, mob/living/died, gibbed)
+	SIGNAL_HANDLER
+	if(!IsContained()) // If it's breaching right now
+		return FALSE
+	if(!ishuman(died))
+		return FALSE
+	if(died.z != z)
+		return FALSE
+	if(!died.mind)
+		return FALSE
+	datum_reference.qliphoth_change(-1) // One death reduces it
+	return TRUE
+
+
+
+//Okay, but here's the life stuff
+/mob/living/simple_animal/hostile/abnormality/eris/Life()
+	..()
+	if(IsContained())
+		return
+	healpulse()
+
+//Okay, but here's the patrolling stuff
+/mob/living/simple_animal/hostile/abnormality/eris/patrol_select()
+	var/list/target_turfs = list() // Stolen from Punishing Bird
+	for(var/mob/living/carbon/human/H in GLOB.human_list)
+		if(H.z != z) // Not on our level
+			continue
+		if(H.stat >= SOFT_CRIT) // prefer the near dead.
+			continue
+		target_turfs += get_turf(H)
+
+	var/turf/target_turf = get_closest_atom(/turf/open, target_turfs, src)
+	if(istype(target_turf))
+		patrol_path = get_path_to(src, target_turf, /turf/proc/Distance_cardinal, 0, 200)
+		return
+	return ..()
+
+
+//Okay, but here's the attacking stuff
+/mob/living/simple_animal/hostile/abnormality/eris/CanAttack(atom/the_target)
+	..()
+	if(!ishuman(the_target))
+		return FALSE
+	var/mob/living/H = the_target
+	if(H.stat >= SOFT_CRIT)
+		return TRUE
+	return FALSE
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/abnormality/eris/CanAttack(atom/the_target)
	if(!ishuman(the_target))
		return FALSE
	var/mob/living/H = the_target
	if(H.stat >= SOFT_CRIT)
		return TRUE
	return ..()
```
OR
```suggestion
/mob/living/simple_animal/hostile/abnormality/eris/CanAttack(atom/the_target)
	if(!ishuman(the_target))
		return FALSE
	var/mob/living/H = the_target
	if(H.stat >= SOFT_CRIT)
		return TRUE
	return FALSE
```
Depending on if you care about the original CanAttack proc or not. Currently you're just running the previous proc without a care for it's value.